docs/pages/choose-an-edition/teleport-cloud/ips.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Public IP Address Allowlist
+description: Restrict outbound network connections from your infrastructure to cloud-hosted Teleport Enterprise.
+tocDepth: 3
+---
+
+Teleport Agents connected to Teleport Enterprise Cloud must be allowed to connect to the following IP addresses in order to join the Teleport cluster.
```

Comment:
```suggestion
Teleport Agents connected to managed Teleport Enterprise accounts must be allowed to connect to the following IP addresses in order to join the Teleport cluster.
```

Reply:
Went with "cloud-hosted" for consistency with the other wording. It's important that self-hosted customers understand that this is not necessary (and less secure) for their setup.